packages/app/src/main/applicationManager.ts

```diff
@@ -236,6 +237,10 @@ export class ApplicationManager {
 
   quit = (): void => {
     this.isQuitting = true;
+    this.windows.main.destroy();
+    this.windows.web.destroy();
+    this.windows.settings.destroy();
+    this.windows.about.destroy();
     app.quit();
```

Comment:
What about command exit? https://www.electronjs.org/docs/latest/api/app#appexitexitcode 
I think the app.quit is "soft" quit with events and such. The exit should 100% kill everything?